src/pcrlock/pcrlock.c

```diff
@@ -4724,6 +4724,16 @@ static int verb_make_policy(int argc, char *argv[], void *userdata) {
         return make_policy(arg_force, arg_recovery_pin);
 }
 
+static int verb_can_make_policy(int argc, char *argv[], void *userdata) {
+        _cleanup_(tpm2_context_unrefp) Tpm2Context *tc = NULL;
+        int r = tpm2_context_new(NULL, &tc);
```

Comment:
in the systemd code base `int r;` is special, we always define it separately at the top of the function, since it's our primary place to put error codes in. i.e. change to:

```c
int r;

r = tpm2_context_new(…);
```